src/test/java/seedu/address/logic/parser/CountryNoteAddCommandParserTest.java

```diff
@@ -35,23 +42,34 @@ public void parse_withPreamable_throwsParseException() {
     @Test
     public void parse_hasCountryNoNote_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse(" c/SG random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/SG t/random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/SG t/"));
     }
 
     @Test
     public void parse_invalidCountryHasNote_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse(" c/Z Z nt/random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/Z Z t/a nt/random string"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/Z Z t/ nt/"));
     }
 
     @Test
     public void parse_validCountryNoNote_throwsParseException() {
         assertThrows(ParseException.class, () -> parser.parse(" c/MY "));
+        assertThrows(ParseException.class, () -> parser.parse(" c/MY t/a"));
+        assertThrows(ParseException.class, () -> parser.parse(" c/MY nt/ t/"));
     }
 
     @Test
     public void parse_validCountryHasNote_equalsExpected() {
         try {
-            CountryNoteAddCommand expected = new CountryNoteAddCommand(
-                    new CountryNote("random string", new Country("SG")));
+            CountryNote countryNote = new CountryNote("random string", new Country("SG"));
+            CountryNoteAddCommand wrongCommand = new CountryNoteAddCommand(countryNote);
+            assertNotEquals(wrongCommand, parser.parse(" c/SG nt/random string"));
```

Comment:
Should this be in a separate test function?

Reply:
Yup, think so.